cmd/server/main.go

```
@@ -22,13 +23,20 @@ func main() {
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "could not load environment: %w\n", err)
+		fmt.Fprintf(os.Stderr, "could not load environment: %s\n", err)
 		os.Exit(1)
 	}
 
+	notificator, err := notificator.New()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "could not create notificator: %s\n", err)
+		os.Exit(1)
+	}
+	notificator.Message(ctx)
```

golangci-lint (1.62.2)

Error return value of `notificator.Message` is not checked

(errcheck)